intra_process_demo/test/two_node_pipeline.regex

```diff
@@ -0,0 +1,2 @@
+Published message with value: (\d+), and address: (0x.*)
```

Comment:
I think you could match the address with something like `0x([a-z]|\d)+` but I haven't tested it.